### DISCUSSION TITLE
where are element and node forces stored in the simulation?

### DISCUSSION THREAD

**Discussion by unknownError** [adtzlr/trusspy#23] [Q&A]
Hi! I was easily managing to get trusspy running on linux - thanks a lot :) I'm trying to simulate some pretty simple static structures and I want to read out the individual element and reaction forces. I looked into a few of the object's properties - however the complexity of the data structure was a bit overwhelming and didn't find the forces :)

could you give me a hint where to look for the reaction forces for each node and the forces inside each element?

Martin

**Comment by adtzlr** [comment]
Hi Martin,

please have a look at the **Example NTA-A**, Section **Verify the results**:

https://adtzlr.github.io/trusspy/examples/eNTA-A.html#verify-the-results 

Hope that helps!

Best regards,
Andreas

**Comment by unknownError** [comment]
Thanks, and yes and no. I'm still not entirely sure how to read the actual forces/stresses. I got the impression they are scaled by a factor? I will have to explore that a little more still :) I'll get back to you if I have further questions after that. Nice Regards, Martin

**Comment by adtzlr** [comment]
Please post a minimal working example and I'll help you if you're still interested.
